Client/src/Pages/Settings/index.jsx

```diff
@@ -37,10 +37,30 @@ const Settings = ({ isAdmin }) => {
   const [form, setForm] = useState({
     ttl: checkTTL ? (checkTTL / SECONDS_PER_DAY).toString() : 0,
   });
+  const [version,setVersion]=useState("");
```

Comment:
Let's provide a default value here in case network request fails, perhaps "unknown" for now is good.